src/components/helpwidget/helpwidget.jsx

```diff
@@ -0,0 +1,119 @@
+const FormattedMessage = require('react-intl').FormattedMessage;
+const injectIntl = require('react-intl').injectIntl;
+const intlShape = require('react-intl').intlShape;
+const bindAll = require('lodash.bindall');
+const connect = require('react-redux').connect;
+const PropTypes = require('prop-types');
+const React = require('react');
+
+const Button = require('../forms/button.jsx');
+/**
+ * Footer link that opens Freshdesk help widger
+ */
+class HelpWidget extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleOpenWidget',
+            'openPopup'
+        ]);
+    }
+    componentDidMount () {
+        // don't add the script to the page more than once
+        if (document.getElementById('helpwidgetscript') === null) {
+            const script = document.createElement('script');
+            script.id = 'helpwidgetscript';
+            script.src = 'https://widget.freshworks.com/widgets/4000000089.js';
+            script.async = true;
+            script.defer = true;
+            script.onload = () => this.scriptLoaded();
+
+            document.body.appendChild(script);
+            window.fwSettings = {
+                widget_id: 4000000089,
+                locale: this.props.intl.locale
+            };
+        }
+    }
+    scriptLoaded () {
+        // freshdesk widget embed code
+        /* eslint-disable */
+        !(function(){if("function"!=typeof window.FreshworksWidget){var n=function(){n.q.push(arguments)};n.q=[],window.FreshworksWidget=n}}())
+        /* eslint-enable */
+        // don't show the Freshdesk button
+        window.FreshworksWidget('hide', 'launcher');
+        window.FreshworksWidget('setLabels', {
+            fr: {
```

Comment:
Why only French?

Reply:
Localization is just a placeholder right now, and I used French as the example.